### PR TITLE
Frogs3.0.0

### PR DESCRIPTION
**Pull request by ValentinMarcon**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

(Poke @mariabernard @oinizan )

### CODE REVIEW

File: recipes/frogs/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,67 @@
...
+    md5: 1a72f44e82517b6839a82cd677b98858 
+  - url: https://github.com/mahendra-mariadassou/phyloseq-extended/archive/v{{ phyloseq_extended_version }}.tar.gz
+    md5: 1ec072751503eebb791e876a60433120
+    folder: lib/external-lib
 
 requirements:
   host:
     - python
-    - rdptools >=2.0.2
+    - rdptools 2.0.2
```
Comment (bgruening) [inline recipes/frogs/meta.yaml]: if possible do not pin that strict and do not pin mpi or openmpi we do this globally

Comment (ValentinMarcon) [reply recipes/frogs/meta.yaml]: Ok so I removed that dependencies

File: recipes/frogs/meta.yaml [hunk 2]
```diff
@@ -1,53 +1,62 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 3
+  number: 0
   skip: true  # [py3k]
 
+# 1) Uplaod FROGS source code
+# 2) Add phyloseq-extented lib from Mahendra Mariadassou to FROGS (lib/external-lib) :
```
Comment (epruesse) [inline recipes/frogs/meta.yaml]: Do we need those comments?

File: recipes/frogs/meta.yaml [hunk 3]
```diff
@@ -1,53 +1,62 @@
...
 source:
-  url: https://github.com/oinizan/FROGS/archive/v{{ version }}.tar.gz
-  md5: da41b0950a750be2038cefa717edf9df
+  - url: https://github.com/geraldinepascal/FROGS/archive/v{{ version }}.tar.gz 
+    md5: 1a72f44e82517b6839a82cd677b98858 
+  - url: https://github.com/mahendra-mariadassou/phyloseq-extended/archive/v{{ phyloseq_extended_version }}.tar.gz
+    md5: 1ec072751503eebb791e876a60433120
+    folder: lib/external-lib
```
Comment (epruesse) [inline recipes/frogs/meta.yaml]: Is there a way to have phyloseq-extended as a separate package?

File: recipes/frogs/meta.yaml [hunk 4]
```diff
@@ -1,53 +1,62 @@
...
-    - swarm >=2.1.13
-    - vsearch >=2.4.3
-    - blast >=2.2.31
-    - rdptools >=2.0.2
+    - perl-io-gzip 0.20
+    - flash 1.2.11
+    - cutadapt 1.18
+    - swarm 2.2.2
+    - vsearch 2.9.1
+    - blast 2.7.1
```
Comment (epruesse) [inline recipes/frogs/meta.yaml]: Those tight pins can make it difficult to find a solution later on. Wouldn't e.g. `blast 2.7.*` suffice?

File: recipes/frogs/meta.yaml (hunk 3 above)
Comment (ValentinMarcon) [reply recipes/frogs/meta.yaml]: Not really because this is really specific to Frogs

File: recipes/frogs/meta.yaml (hunk 4 above)
Comment (ValentinMarcon) [reply recipes/frogs/meta.yaml]: We have chosen to pin **all** the dependencies version to ensure we will **always** have the same results with this version of FROGS

File: recipes/frogs/meta.yaml (hunk 2 above)
Comment (ValentinMarcon) [reply recipes/frogs/meta.yaml]: Why not (except for the mistyping) ?

**Comment by ValentinMarcon** [comment]
Hello,

According to your advices @bgruening we decided to unpin the dependencies.
Hope this time all the recipe will be fine 🤞 
Cheers

( @epruesse  @mariabernard )

**Pull request merged**